tripl/tripl.py

```diff
@@ -29,29 +31,32 @@ def log(name, value):
     pprint.pprint(value)
     print("\n")
 
+
 def some(xs, default=None):
-    "return some thing from the set, or None if nothing"
-    if isinstance(xs, (str, unicode, int, float, bool, dict, Entity, uuid.UUID)):
+    """return some thing from the set, or None if nothing"""
+    if isinstance(xs, (str, int, float, bool, dict, Entity, uuid.UUID)):
```

Comment:
Please note that I removed `unicode` from the list, because that does not exist in Python 3. This might break Python 2.

Reply:
Hmm... this may be a problem. I'll look into it. Stupid that a `unicode` isn't also a `str`. Thanks.

Reply:
In Python 2 `unicode` is for strings and `str` for binary data. In Python 3 `str` is for strings and `bytes` for binary data.